linux_os/guide/system/permissions/files/file_permissions_unauthorized_world_writable/bash/shared.sh

```diff
@@ -0,0 +1,3 @@
+# platform = multi_platform_rhel
+
+(cd / && find . -type f -perm -002 -not -path "./selinux/*" -not -path "./proc/*" -not -path "./sys/*" -not -path "./cgroup/*" -exec chmod -v o-w {} +)
```

Comment:
This should be:

`find / -xdev -type f -perm -002 -exec chmod o-w {} \;`

Reply:
I agree:

- `xdev` says "don't search directories that are mounted from different devices than the root's device", which is a simpler version of those `-not -path "..."`.
- find is position-independent when used with absolute paths.